kedro/config/omegaconf_config.py

```diff
@@ -170,6 +175,9 @@ def __getitem__(self, key) -> dict[str, Any]:
             )
         patterns = [*self.config_patterns[key]]
 
+        if key == "globals":
+            # "runtime_params" resolver is not allowed in globals.
+            OmegaConf.clear_resolver("runtime_params")
```

Comment:
Does that means running `config_loader["global"]` twice could get different results? User may use `config_loader["global"]` explicitly to perform logging etc. https://kedro-org.slack.com/archives/C03RKP2LW64/p1695380623784409

Reply:
I don't think running `config_loader["globals"]` would return anything different each time, it just turns off the `runtime_params` resolver.